### PR TITLE
Update recommended-settings-for-eop-and-office365-atp.md

### PR DESCRIPTION
**Pull request by andypunt**
Adding links to ORCA.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/recommended-settings-for-eop-and-office365-atp.md [hunk 1]
```diff
@@ -27,6 +27,9 @@ Although we empower security administrators to customize their security settings
 
 This topic describes these Microsoft-recommended settings to help protect your Office 365 users.
 
+> [!TIP]
+> There is a new PowerShell Module that you can download called the Office 365 Advanced Threat Protection Recommended Configuration Analyzer (ORCA) that helps determine some of these settings. When run as an admin in your tenant, Get-ORCAReport will help generate an assessment of the anti-spam, anti-phish, and other message hygiene settings. You can download this module at https://www.powershellgallery.com/packages/ORCA/.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/recommended-settings-for-eop-and-office365-atp.md:31]:
```suggestion
> There is a new PowerShell module called the Office 365 Advanced Threat Protection Recommended Configuration Analyzer (ORCA) that you can get to determine some of these settings. When run as an admin in your tenant, the `Get-ORCAReport` cmdlet generates an assessment of the anti-spam, anti-phish, and other message hygiene settings. You can download this module from the [PowerShell Gallery](https://www.powershellgallery.com/packages/ORCA).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - suggestion only.

**Comment by mypil** [comment]
@andypunt  - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Review by denisebmsft — APPROVED** [review]
Thanks @andypunt

**Pull request merged**